### DISCUSSION TITLE
Multi tenant support

### DISCUSSION THREAD

**Discussion by jbramamoorthy** [cookiecutter/cookiecutter-django#3852] [Ideas]
## Description

What are you proposing? How should it be implemented?

Django application requires multi-tenant functionality for enterprise saas-based applications.

## Rationale

Why should this feature be implemented?

Saas application feature

**Comment by luzfcb** [comment]
Multi-tenant applications are interesting, and it would be nice to have a ready-made and extensively tested template to be a good starting point.
That said, the subject can be big and complex, and I (and I believe the other maintainers) will not have the free time to create it and maintain it indeterminately, meaning the cost will be high for us.

Remember, none of the maintainers receive any remuneration to maintain this project, and we do it in our spare time.

That said, you can do your own investigations:

https://books.agiliq.com/projects/django-multi-tenant/en/latest/

https://github.com/schinckel/django-occupation
https://github.com/django-tenants/django-tenants
https://github.com/citusdata/django-multitenant

https://djangopackages.org/grids/g/multi-tenancy/


if anyone thinking of creating and maintaining a multi-tenant version of cookiecutter-django, we will be happy to point it out a link in our readme.
